hack/boilerplate/check-boilerplate.go

```diff
@@ -116,14 +116,9 @@ func main() {
 		os.Exit(1)
 	}
 
-	hasErr := false
 	for _, filePath := range os.Args[1:] {
 		if err := verifyFile(filePath); err != nil {
 			fmt.Printf("error validating %q: %v\n", filePath, err)
```

Comment:
This message should be printed in stderr, while now it is streamed to stdout, which seems to be incorrect. WDYT? @SataQiu

Reply:
`pouchcontainer/pouchlinter:v0.2.1` has no GCC installed. 
Emm, I don't know where GCC is used, some cgo code?
Update `pouchcontainer/pouchlinter:v0.2.1` or create a new job using `circleci/golang:1.12.6` ?
WDYT? @allencloud

Reply:
We plan to replace gometalinter with golangci-lint and golangci-lint needs gcc dependency. Latest pouchcontainer/pouchlinter:v0.2.2 has installed gcc if you need it.

Reply:
OK! Let's bump to `pouchcontainer/pouchlinter:v0.2.2`. :laughing:

Reply:
But in `pouchcontainer/pouchlinter:v0.2.2` we removed gometaliner. We need to update circle-ci part to migrate to golangci-lint first. So I will open a PR tonight and after that you can work on this PR. Is it ok? I will finish it ASAP.

Reply:
OMG. 
`pouchcontainer/pouchlinter:v0.2.2` has no `gometalinter` installed. :(